site/docs/v1/tech/shared/_scim-limits.adoc

```diff
@@ -1,20 +1,15 @@
-The current SCIM implementation does not support:
-
-* Bulk operations
+The current SCIM implementation does not support bulk operations.
 
 Bulk API operations are available via link:/docs/v1/tech/apis/[FusionAuth APIs], but are not currently supported via SCIM compatible endpoints. If this functionality is required, please file a https://github.com/fusionauth/fusionauth-issues/issues[GitHub issue] explaining your use case.
 
 The list operation is limited to 10,000 users if you are using the Elasticsearch search engine. This https://github.com/FusionAuth/fusionauth-issues/issues/494[GitHub issue] discusses this limitation.
 
-You cannot create a FusionAuth Registration for a user provisioned with SCIM. If you need to register the user for one or more Applications, you have a few options. You may:
-
-* use the link:/docs/v1/tech/lambdas/scim-user-request-converter[SCIM User Request Converter Lambda].
-* use the link:/docs/v1/tech/events-webhooks/events/user-create-complete[`user.create.complete` webhook].
-
-Either works. The correct choice depends on if you want to do the processing within FusionAuth using a lambda or using an external server.
-
-While it is not enforced, or a hard limitation, it is recommended to configure at most one SCIM server per tenant.
+You cannot create a FusionAuth Registration for a user provisioned with SCIM directly during the provisioning process. See link:/docs/v1/tech/core-concepts/scim#adding-registrations[Adding Registrations] for workarounds.
 
-While you can have multiple SCIM servers per configured within one tenant, at the end of the day you only have one user data store per tenant.
+While it is not enforced, or a hard limitation, it is recommended to configure at most one SCIM server per tenant. While you can have multiple SCIM servers configured to provision users in one FusionAuth tenant, at the end of the day you only have one user data store in each tenant. Conflicts with creation, updates and deletion could happen; they won't be automatically resolved by FusionAuth. If you need to use multiple SCIM servers, ensure your clients can handle such conflicts.
```

Comment:
From the doc:
> FusionAuth can consume SCIM formatted requests and act as the provisioning server in a SCIM client/server environment.

I think the user of "server" here is referring to a customer's server, but that server is acting as the _SCIM client_. Can we use "client" here instead of "server"?

> ensure your clients can handle such conflicts.

Does a _SCIM client_ also retrieve and consume data from the _SCIM server_? Or does "clients" here refer to anything that queries FusionAuth for user data?